workshops/3D Heatmap/README.md

```diff
@@ -0,0 +1,204 @@
+author: '@shivesh01'
+image: 'https://cloud-a08hob7s0.vercel.app/0workshop1.gif'
+---
+
+
+# Volcano Heat Map
+
+![Demo](https://cloud-94iqxy8lo.vercel.app/0volcano.gif)
+
+[Click here to see a working demo of today's project](https://repl.it/@ShiveshSingh/Volcano-Heatmap#main.py)
+
+In our workshop, you will use Python programming language and libraries to create a stunning heat map from scratch. And you also learn about 3D heatmaps. Our project is about collecting data then sort, arrange and present the data in a 3D model. We know 3D modeling has changed the way we design for the better.
+
+![How to present data](https://cloud-3ifpxv546.vercel.app/0image.png)
+
+
+# Getting started
+
+You need to set up a repl account or run this program on your Python idle. It requires some basic knowledge of programming. We have designed this workshop as sweet and simple for you. It will only take 15-20 minutes to complete the workshop. 
+
+## Setting up
+
+- Repl is an online, instant development environment to learn, build, collaborate, and host your project. Show you don’t have to waste time while setting up a development environment. Creating an account on repl is super easy.
+
+[Create a repl](https://repl.it/signup)
+
+
+![repl website](https://cloud-73h0sldam.vercel.app/0screenshot_2020-12-25_at_23.03.53.png)
+
+- Create a repl by clicking **+ New repl** and choose programming language **PYTHON** and named it as you want and **create repl**
+
+- Download the dataset from below, drag and drop them to the folder you are working on repl.\
+[volcano.csv](https://cloud-8ycpvzexa.vercel.app/0volcano.csv)
+
+
+- Rename the file as volcano.csv and place it in the repl folder in which you are working.
+
+- The setup is complete. ✅.
+
+**Advantage of working on repl**
+
+*When we program in repl, it automatically imports libraries and dependencies with a simple command. So, you just require a repl account or any python idle get started.*
+
+![Library](https://cloud-1th3ydnib.vercel.app/1workshop_library.gif)
+
+## Pandas🐼
+Pandas is a popular Python-based toolkit. It presents a diverse range of utilities like converting an entire data table into a NumPy matrix array and much more. That makes pandas a trusted ally in data science and machine learning.
+
+## Numpy🔢
+We use NumPy to work with arrays to perform a large set of mathematical operations. An array is a data structure consisting of a collection of elements, each identified by at least one array index or key.
+
+
+## Matplotlib📊
+A picture is worth a thousand words, and with Python’s matplotlib library, fortunately, it takes far less than a thousand words of code to create a production-quality graphic.
+
+## Toolkits🧰
+Toolkits are collections of application-specific functions that extend Matplotlib.
+
+## Seaborn䷀
+Seaborn library is one of the rarest earth metals. Seaborn, which provides a variety of visualization patterns, uses fewer syntax and decent themes.
```

Comment:
I don't really understand the final three, they don't fully explain the libraries.